docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1602 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    num_rings = len(geometry.polygons.ring_offset)
</pre><p>Some variables are unused in this cell <code>num_rings</code>,<code>points_x</code>,<code>points_y</code>,<code>points_offsets</code>        </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Thanks for catching this.